### ISSUE TITLE
Making searches by "Administrative Area" reliable

### ISSUE THREAD

**Issue by albenson-usgs** [gbif/hosted-portals#130]
I noticed [an issue](https://github.com/gbif/portal-feedback/issues/3364) come into the GBIF portal feedback and wondered if a user would experience the same results in the hosted portals. The answer is yes.
![image](https://user-images.githubusercontent.com/7586150/114467817-b39a2700-9ba7-11eb-8dc6-a83b9f7de785.png)

For users of the current BISON portal it's important for them to be able to select observations by state or county. I had thought the Administrative Areas/GADM search filter would provide this functionality but I'm not sure it will provide the same functionality.

I have two concerns with what's being displayed when I try searching for Marion County, Texas using Administrative Areas:
1. Texas is not displayed so it cannot be selected and you cannot simply type "Marion Texas" and click enter.
2. The options that come back include areas outside of the United States. It would be nice if there was a way to limit what the user can select based on the geographic scope of the portal. (I realize this may have to go into the future enhancements bucket).

I also tried using the Keyword Search and typing "Marion Texas" but the results that come back are from throughout the US. https://hp-bison.gbif-staging.org/data?filter=eyJtdXN0Ijp7InEiOlsiTWFyaW9uIFRleGFzIl19fQ%3D%3D&view=map

Would it be possible to incorporate consecutive filtering so that first you select Country= "United States", then select State = "Texas", then select County = "Marion"?

**Comment by tucotuco** [comment]
I agree that the current method is problematic. Imagine a really common
case, such as Santa Cruz, where there are about 125 at admin level 2 alone.
Across all admin levels I think Santa Cruz is second only to San Isidro,
with more than a thousand combos.

**Comment by MortenHofft** [comment]
Same as https://github.com/gbif/portal-feedback/issues/3044
Any API changes will benefit all.
UI changes will have to be implemented twice (once for GBIF.org + once for the hosted portals)

**Comment by albenson-usgs** [comment]
The closing of issue #136 doesn't fix this issue, right? If it does, a demonstration on how to do it would help.

**Comment by MortenHofft** [comment]
No it does not

**Comment by MortenHofft** [comment]
I've changed the threshold to include more results. It is more of a bandaid than a solution, but it does "solve" the issues I have seen reported so far. But it leaves it to the user to shift through more results.
I have also suggested some specific improvements to the API that I believe would solve it more properly
https://github.com/gbif/portal-feedback/issues/3044#issuecomment-924766244

Having a way to browse location top down (country > state > county in us context) would still be a nice feature, but I believe better search could take us a long way.

**Comment by MortenHofft** [comment]
I'm closing this as the issue is already exists in a better place. If it is solved in the API it will be solved in the hosted portals
